sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -667,6 +667,7 @@ export class AssessmentSequentialProcessingService {
         .andWhere("assessment.offering IS NOT NULL")
         .andWhere("application.student.id = :studentId")
         .andWhere("application.programYear.id = :programYearId")
+        .andWhere("application.applicationStatus != :applicationEditedStatus")
```

Comment:
The logic looks good here considering the status flip on ministry approval. But on the new change request application, I believe we need a logic update regarding the original assessment study start date. With current change request, the original assessment study start date must be the parent application's original assessment study start date right?

Reply:
The method `getOutstandingAssessmentsForStudentInSequence` is already used for edited applications before COE and reassessments. What is the issue that the change request would introduce if this method is not changed?

Reply:
Thanks for the chat. As per our conclusion, we are good to proceed as it is. Nice conversation, though.